merkle/common_test.go

```diff
@@ -42,7 +41,7 @@ func setupProofTestCases() ([]ProofTestCases, error) {
 	var merkleProofCases []ProofTestCases
 	for i := 0; i < len(maxCase); i++ {
 		var leavesHashes [][]byte
-		var Leaves []types.Leaf
+		var Leaves Leaves
```

Comment:
```suggestion
		var leaves Leaves
```

Reply:
fixed in [this](https://github.com/ComposableFi/go-merkle-trees/pull/22/commits/2e360cdbbfc6c3e12c3c206b0c3b3d92e91c702c)